src/components/Proposal/Create/CreateProposal.scss

```diff
@@ -15,7 +15,7 @@
   left: 50%;
   transform: translate(-50%, -50%);
   box-shadow: $shadow-3;
-  height: calc(100% - 100px);
+  height: calc(80%);
```

Comment:
Does this change work consistently on all widths (I would check mobile too even though it is lower priority)?

Reply:
Why wouldn't it? Not sure what you mean.

Reply:
You have replaced a constant 100px with a variable 20%

Reply:
oh, and sorry I should've said "all heights" not "all widths"